INSTALL.md

```diff
@@ -1,75 +1,72 @@
+4. Install Elf-simulations.
 
 ```bash
 python -m pip install --upgrade pip
-python -m pip install -e .[with-dependencies]
+python -m pip install --upgrade -e ".[with-dependencies]"
 ```
 
 If you intend to improve the documentation, then you must also install the packages:
 
 ```bash
-python -m pip install -e .[with-dependencies,docs]
+python -m pip install --upgrade -e ".[with-dependencies,docs]"
 ```
 
-- `pyenv install 3.9` You should now see the correct version when you run `pyenv versions`.
-- `pyenv local 3.9` This command creates a `.python-version` file in your current directory. If you have pyenv active in your environment, this file will automatically activate this version for you.
+An explanation of what the above steps do:
+- `pyenv install 3.10` You should now see the correct version when you run `pyenv versions`.
+- `pyenv local 3.10` This command creates a `.python-version` file in your current directory. If you have pyenv active in your environment, this file will automatically activate this version for you.
 - `python -m venv .venv` This will create a `.venv` folder in your repo directory that stores the local python build & packages. After this command you should be able to type which python and see that it points to an executable inside `.venv/`.
-- `python -m pip install -e .[with-dependencies]` This installs elfpy locally such that the install updates automatically any time you change the source code. This also installs all dependencies defined in `pyproject.toml`.
-
-Finally, you can test that everything is working by calling:
-`PYTEST_DISABLE_PLUGIN_AUTOLOAD=1 python -m pytest tests/`
+- `python -m pip install --upgrade -e ".[with-dependencies]"` This installs elfpy locally such that the install updates automatically any time you change the source code. This also installs all dependencies defined in `pyproject.toml`.
 
-## Apeworks and Contract Integration
+## Working with smart contracts (optional)
 
 We run several tests and offer utilities that depend on executing Hyperdrive solidity contracts. This is not required to use elfpy.
 
 NOTE: The Hyperdrive solidity implementation is currently under security review, and thus is not available publicly.
 The following instructions will not work for anyone who is not a member of Delv.
 
-First, [install Forge](https://github.com/foundry-rs/foundry#installatio://github.com/foundry-rs/foundry#installation).
+5. Set up smart contracts
 
-Next, to use apeworx with elfpy, clone and sym link the hyperdrive repo, into `hyperdrive_solidity/`, i.e.:
+Clone and sym link the hyperdrive repo, into `hyperdrive_solidity/`, i.e.:
 
 ```bash
 git clone https://github.com/delvtech/hyperdrive.git ../hyperdrive
 ln -s ../hyperdrive hyperdrive_solidity
 ```
 
-then run:
+6. Install Hyperdrive pre-requisites
 
-```bash
-ape plugins install .
-ape compile
-```
+Complete the steps in [Hyperdrive's Pre-requisites section](https://github.com/delvtech/hyperdrive#pre-requisites).
```

Comment:
They also need to follow the steps in the `build` section, correct? Since you dropped `ape compile` from the commands?

Reply:
Ape compiles whenever it senses the need, or a mild breeze rolls through, so it's not a required install step.